test/e2e/opm_test.go

```diff
@@ -44,6 +44,7 @@ var (
 	indexImage1 = "quay.io/olmtest/e2e-index:" + indexTag1
 	indexImage2 = "quay.io/olmtest/e2e-index:" + indexTag2
 	indexImage3 = "quay.io/olmtest/e2e-index:" + indexTag3
+	indexImage4 = "quay.io/operatorhubio/catalog:latest"
```

Comment:
Can we use one of the test indexes that we already build rather than rely on an external dependency that we don't control?

Reply:
changed it to use `indexImage2`